CoolestMovieAPI/CoolestMovieAPI/Controllers/MoviesController.cs

```diff
@@ -25,86 +25,244 @@ public class MoviesController : ControllerBase
         public MoviesController(IMovieRepository movieRepository)
         {
             _movieRepository = movieRepository;
-           
         }
        
+
+        [HttpGet]
+        public async Task<ActionResult<IList<Movie>>> GetAll()
```

Comment:
This comment is more for the future, but I think I will bring it up here since you are going to start researching AutoMapper. But we will need to change the return type to MovieDTO instead of the Movie model.